src/renderer/services/binance/utils.ts

```diff
@@ -58,18 +56,16 @@ export const bncSymbolToAsset = (symbol: string): O.Option<Asset> =>
 export const bncSymbolToAssetString = (symbol: string) => `${AssetBNB.chain}.${symbol}`
 
 export const getWalletBalances = (balances: Balances): AssetsWithBalance =>
-  balances.reduce((acc: AssetsWithBalance, { symbol, free, frozen }: Balance) => {
+  balances.reduce((acc: AssetsWithBalance, { symbol, free }: Balance) => {
```

Comment:
just as a suggestion seems we could use `filterMap` from 'fp-ts/Array' here. just to increase readability